libraries/mbed/targets/cmsis/TARGET_Freescale/TARGET_K22F/cmsis_nvic.c

```diff
@@ -31,14 +31,14 @@
 #include "cmsis_nvic.h"
 
 #define NVIC_RAM_VECTOR_ADDRESS (0x1FFF0000)  // Vectors positioned at start of RAM
-#define NVIC_FLASH_VECTOR_ADDRESS (0x0)       // Initial vector position in flash
+#define NVIC_FLASH_VECTOR_ADDRESS (__vector_table)       // Initial vector position in flash
```

Comment:
Is this macro used anywhere? __vector_table is not defined for ARMCC. This I believe is also for that KL46 which we could fix